src/applications/_mock-form-ae-design-patterns/app-entry.jsx

```diff
@@ -7,9 +7,16 @@ import routes from './routes';
 import reducer from './reducers';
 import manifest from './manifest.json';
 
+import coeReducer from './patterns/pattern2/TaskRed/shared/reducers';
+
+const combinedReducers = {
+  ...reducer,
+  certificateOfEligibility: coeReducer.certificateOfEligibility,
+};
+
```

Comment:
added the COE reducer here so that it can be hooked into for the form. This _could_ potentially be loaded in async, but the startApp function does not support that... yet